pegasus/sites.v3/code.org/styles_min/030-font-awesome-min.css

```diff
@@ -1,4 +1,4 @@
 /*!
  *  Font Awesome 4.6.3 by @davegandy - http://fontawesome.io - @fontawesome
  *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
- */@font-face{font-family:'FontAwesome';src:url('/fonts/fontawesome-webfont.eot?v=4.6.3');src:url('/fonts/fontawesome-webfont.eot?#iefix&v=4.6.3') format('embedded-opentype'),url('/fonts/fontawesome-webfont.woff2?v=4.6.3') format('woff2'),url('/fonts/fontawesome-webfont.woff?v=4.6.3') format('woff'),url('/fonts/fontawesome-webfont.ttf?v=4.6.3') format('truetype'),url('/fonts/fontawesome-webfont.svg?v=4.6.3#fontawesomeregular') format('svg');font-weight:normal;font-style:normal}.fa{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.fa-lg{font-size:1.33333333em;line-height:.75em;vertical-align:-15%}.fa-warning:before{content:"\f071"}.fa-chevron-up:before{content:"\f077"}.fa-chevron-down:before{content:"\f078"}.fa-twitter:before{content:"\f099"}.fa-facebook:before{content:"\f09a"}.fa-caret-down:before{content:"\f0d7"}.fa-caret-up:before{content:"\f0d8"}.fa-instagram:before{content:"\f16d"}.fa-tumblr:before{content:"\f173"}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0, 0, 0, 0);border:0}.fa-user:before{content:"\f007"}
+ */@font-face{font-family:'FontAwesome';src:url('/fonts/fontawesome-webfont.eot?v=4.6.3');src:url('/fonts/fontawesome-webfont.eot?#iefix&v=4.6.3') format('embedded-opentype'),url('/fonts/fontawesome-webfont.woff2?v=4.6.3') format('woff2'),url('/fonts/fontawesome-webfont.woff?v=4.6.3') format('woff'),url('/fonts/fontawesome-webfont.ttf?v=4.6.3') format('truetype'),url('/fonts/fontawesome-webfont.svg?v=4.6.3#fontawesomeregular') format('svg');font-weight:normal;font-style:normal}.fa{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.fa-lg{font-size:1.33333333em;line-height:.75em;vertical-align:-15%}.fa-warning:before{content:"\f071"}.fa-chevron-up:before{content:"\f077"}.fa-chevron-down:before{content:"\f078"}.fa-twitter:before{content:"\f099"}.fa-facebook:before{content:"\f09a"}.fa-caret-down:before{content:"\f0d7"}.fa-caret-up:before{content:"\f0d8"}.fa-instagram:before{content:"\f16d"}.fa-tumblr:before{content:"\f173"}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0, 0, 0, 0);border:0}.fa-user:before{content:"\f007"}.fa-question-circle:before{content:"\f059"}
```

Comment:
I think in the past we've tried to insert these in the order they appear in the source file.